frontend/src/components/sign_up/sign_up.css

```diff
@@ -1,130 +0,0 @@
-:root {
```

Comment:
I'd like to keep this `sign_up` component (including this file and the JS one below) for now until we can get the sign up/login page working, then we can delete the stuff that isn't being used.